### PR TITLE
systemvm: fix keepalived is always restarted when update config for monitor service

### PR DESCRIPTION
**Pull request by ustcweizhou**
## Description

in 4.15, keepalived in redundant VRs keeps restarting every minute.
After debugging, I found it happens when update config for monitor service.
it is because keepalived process is changed in Debian 10.

in Debian 9 (systemvm for 4.14),
```
root@r-1969-VM:~# ps -ef|grep keepalived
root     16324     1  0 09:53 ?        00:00:04 /usr/sbin/keepalived
root     16325 16324  0 09:53 ?        00:00:04 /usr/sbin/keepalived
root     16326 16324  0 09:53 ?        00:00:14 /usr/sbin/keepalived
```

in Debian 10 (systemvm for 4.15), processes end with "--dont-fork"
```
root@r-2040-VM:~# ps -ef|grep keepalived
root      5237     1  0 16:40 ?        00:00:00 /usr/sbin/keepalived --dont-fork
root      5239  5237  0 16:40 ?        00:00:03 /usr/sbin/keepalived --dont-fork
```

## Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

### CODE REVIEW

**Review by DaanHoogland — APPROVED** [review]
code lgtm

File: systemvm/debian/opt/cloud/bin/cs/CsRedundant.py [hunk 1]
```diff
@@ -194,7 +194,7 @@ def _redundant_on(self):
         heartbeat_cron.commit()
 
         proc = CsProcess(['/usr/sbin/keepalived'])
```
Comment (sureshanaparti) [inline systemvm/debian/opt/cloud/bin/cs/CsRedundant.py:196]: @ustcweizhou specifying "/usr/sbin/keepalived" here may not be required as grep is used instead.

I understand this issue is with _keepalived_ process, but on a broader view, other processes might have similar issue later, can this be fixed in _CsProcess.py find()_ ?

Comment (weizhouapache) [reply systemvm/debian/opt/cloud/bin/cs/CsRedundant.py:196]: @sureshanaparti yes, of course.
To be sure that is not regression issue, I choosed a easier way to fix it.

Comment (sureshanaparti) [reply systemvm/debian/opt/cloud/bin/cs/CsRedundant.py:196]: ok @weizhouapache , got it. Note that, if the same issue repeats for other processes later, then _CsProcess.py_ have to be fixed.

**Review by sureshanaparti — APPROVED** [review]

**Comment by weizhouapache** [comment]
@rhtyd @DaanHoogland this needs to be merged asap.

the test failure should be fixed by this pr.

Test | Result | Time (s) | Test File
test_03_create_redundant_VPC_1tier_2VMs_2IPs_2PF_ACL_reboot_routers | Failure | 290.08 | test_vpc_redundant.py

**Comment by DaanHoogland** [comment]
ok @weizhouapache , will run tests once
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test ubuntu kvm-ubuntu keepEnv

**Comment by DaanHoogland** [comment]
@blueorangutan test ubuntu kvm-ubuntu

**Comment by DaanHoogland** [comment]
@blueorangutan test keepEnv

**Comment by DaanHoogland** [comment]
using this PR for some testing seems not fair :(
@blueorangutan test

**Comment by DaanHoogland** [comment]
@blueorangutan test

**Comment by DaanHoogland** [comment]
none of these errors are related, @rhtyd @sureshanaparti or others, can we merge?

**Comment by rhtyd** [comment]
The same PR was sent to 4.14 https://github.com/apache/cloudstack/pull/4384 - should we close this one and merge the other one?

**Pull request closed without merging**